flash/core/model.py

```diff
@@ -91,9 +91,11 @@ def __new__(mcs, *args, **kwargs):
         result = ABCMeta.__new__(mcs, *args, **kwargs)
         if result.required_extras is not None:
             result.__init__ = _requires_extras(result.required_extras)(result.__init__)
-            result.load_from_checkpoint = classmethod(
-                _requires_extras(result.required_extras)(result.load_from_checkpoint.__func__)
-            )
+            load_from_checkpoint = getattr(result, "load_from_checkpoint", None)
```

Comment:
```suggestion
            load_from_checkpoint = getattr(result, "load_from_checkpoint")
```
that is default I think...

Reply:
I think the default behaviour is to raise an AttributeError if the attribute isn't found